public class TryCatchBlockSecondExample {
    public static void main(String args[]){
        try {
            int data=10/0;
        } catch (Exception e) {
            //TODO: handle exception
            System.out.println(10/2);
        }
    }
}
